docs/modules/buffer-equal.md

```diff
@@ -0,0 +1,11 @@
+# `buffer-equal`/`buffer-equals`
+
+[`buffer-equal`](https://www.npmjs.com/package/buffer-equal) and [`buffer-equals`](https://www.npmjs.com/package/buffer-equals) recommends that you switch away from them to native `Buffer.prototype.equals` which has been available since Node.js v0.11.13
+
+# Alternative(s)
+
+## Native `Buffer.prototype.equals`
+
+For most cases, native JavaScript `Promise` is enough.
```

Comment:
Was this a bad copy paste? 👀

I guess you meant native equals rather than promise

Reply:
yeah, sorry, forgot to save